R/module_nested_tabs.R

```diff
@@ -263,7 +265,6 @@ srv_teal_module.teal_module <- function(id,
 
     trigger_module <- reactive({
       req(module_teal_data_validated())
-      req(filtered_teal_data())
```

Comment:
@gogonzo when this is deleted I do not see data on the second module when I select the second module in a simple app

```r
app <- init(
  data = teal_data(iris = iris, mtcars = mtcars),
  modules = modules(example_module("Module 1"), example_module("Module 2"))
)

shinyApp(app$ui, app$server)

```

<img width="671" alt="image" src="https://github.com/user-attachments/assets/f701e515-3c2f-4b44-bdd3-0fcffcfe1ab5">

Reply:
You can revert this line. We can deal with this later this week

Reply:
I think there is something off with `.observe_active_filter_changed` that returns `NULL` on `init` hence

```r
    filtered_teal_data <- srv_filter_data(
      "filter_panel",
      datasets = datasets,
      active_datanames = active_datanames,
      data_rv = data_rv,
      is_active = is_active
    )
```

returns error